k8s/demo/common/ccd/ccd-full-with-pay.yaml

```diff
@@ -65,6 +65,8 @@ spec:
           enabled: false
         ccpay-bulkscanning-api:
           enabled: false
+        ccd:
```

Comment:
ccd is enabled by default from ccpay chart?

Reply:
yes